overrides.json

```diff
@@ -1239,5 +1239,21 @@
         "./dist/mini.umd.js": "./dist/mini.umd.js"
       }
     }
+  },
+  "@vue/reactivity": {
+    "3": {
+      "exports": {
+        "module": "./dist/reactivity.esm-browser.js",
+        "default": "./index.js"
+      }
+    }
+  },
+  "@vue/shared": {
+    "3": {
+      "exports": {
+        "module": "./dist/shared.esm-browser.js",
```

Comment:
This file does not exist actually, there's only the bundler version for 3.3.4 - https://unpkg.com/browse/@vue/shared@3.3.4/dist/.